python/social_bot/envs/grocery_ground.py

```diff
@@ -515,7 +577,11 @@ def _get_camera_observation(self):
         return img
 
     def _get_low_dim_full_states(self):
-        task_specific_ob = self._teacher_task.task_specific_observation()
+        task_specific_ob = np.array([])
+        for task_group in self._teacher.get_task_group():
+            for gro_task in task_group.get_tasks():
+                task_specific_ob = np.append(task_specific_ob,
+                    gro_task.task_specific_observation())
```

Comment:
should put line 580-584 to Teacher

Reply:
Yes, I understand. But not all tasks need specified observation, only tasks inherited from "GroceryGroundTaskBase" have this function defined for now. Should we keep it this way or add abstract "task_specific_observation()" to Task base class?

Reply:
Adding to Task class is better.